content/engineering/nodejs.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Node.js
+sidenav: true
+permalink: /engineering/nodejs/
+tags: engineering
+layout: layouts/page
+eleventyNavigation: 
+  parent: engineering_languages
+  key: Node.js
+  order: 4
+  title: Node.js
+---
+
+## Related topics
+* [JavaScript]({{site.baseurl}}/javascript)
```

Comment:
I'll make a note in the general PR review, but anywhere that has `{{site.baseurl}}` like this is a leftover from Jekyll, and will need to be updated to the correct address.